Failed to build IDE image on zjs-0.4

`make ide ROM=256` is failing to build IDE image on zjs-0.4 branch. 

Error:
```
/opt/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/libexec/i586-zephyr-elfiamcu/gcc/i586-zephyr-elfiamcu/6.2.0/real-ld: zephyr.elf section `initlevel' will not fit in region `ROM'
/opt/zephyr-sdk/sysroots/x86_64-pokysdk-linux/usr/libexec/i586-zephyr-elfiamcu/gcc/i586-zephyr-elfiamcu/6.2.0/real-ld: region `ROM' overflowed by 140 bytes
collect2: error: ld returned 1 exit status
```

Note: If I increase the ROM size, the ARC image doesn't fit in available space.

--- a/src/zjs_common.h
+++ b/src/zjs_common.h
@@ -72,8 +72,7 @@ int zjs_get_ms(void);
 #else  // !DEBUG_BUILD
 #define DBG_PRINT(fmt...) do {} while (0)
 #define ERR_PRINT                                                        \
-    ZJS_PRINT("\n%s:%d %s():\n(ERROR) ", zjs_shorten_filepath(__FILE__), \
-              __LINE__, __func__);                                       \
+    ZJS_PRINT("\n%d:(ERROR) ", __LINE__);                                \
     ZJS_PRINT
 #endif  // DEBUG_BUILD
 

